### DISCUSSION TITLE
Automate and E-Mails

### DISCUSSION THREAD

**Discussion by geekrambo** [Budibase/budibase#1976] [Help] [answered]
Hello All,

I am attempting to utilize the email option in the Automate area to email a row of form data to someone.

I click on the Test Automation and get a test email, but when running the actual automation, I'm not receiving an email.

Are there examples of this automation that I can reference from? I am looking at the handlebars site for reference, but don't know what I'm looking for there.

**Comment by mike12345567** [comment, marked as answer]
Hi @geekrambo - one thing to check is that you've activated your automation, this isn't very clear right now and trips a lot of people up, but in the top right of the automation panel you'll see:
![image](https://user-images.githubusercontent.com/4407001/125423904-f2412090-bdc6-4aef-9d18-fc25535be9a3.png)

If your automation looks like in the picture this means that it hasn't been activated and won't function on its actual trigger, it'll only work in testing - click the play button and it'll work then!

**Reply by geekrambo** [reply]
Thank you, I found the answer accidentally, but very much appreciate the response!

**Reply by geekrambo** [reply]
So, I have another question regarding Automation. How does one utilize the "Created By" column of a table with the mustache tags? I'm trying to set the "Send As" email address in my Automation with {{ trigger.row.CreatedBy }}, but the system keeps sending as the SMTP address I setup in the beginning.

Until I figure out how to get my service desk software to work with REST APIs, I'm having budibase send an email into it. The service desk software sends Auto Replies back to the sender of the email. I'd like those replies to go to whoever created the row in my forms table. Any suggestions?

THANK YOU!

Final note... As a "doesn't do software programming in any way, shape, or form" type of person, I'm quite impressed with the budibase platform! Thank you Thank you Thank you for making something that helps up n00bs look smart n' stuff! :)

**Reply by mjashanks** [reply]
Hey @geekrambo this is not possible at the moment. 

I think what you need to be able to do is something like `{{ trigger.row.CreatedBy.Email }}` - i.e. access the `Email` of the `CreatedBy` user. 

It's something that we are aware of - we do have some work to do around relationships in automations. 

Thanks for the kind words - glad you are enjoying Budibase!
